recipes/molpx/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "molPX" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "f11bb1501195f922de7081198e853a312225d05554b2bb972b62dbb16938c09d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - nglview >=0.6.2.3
+    - pyemma
+    - scikit-learn
+    - notebook
+
+test:
+  imports:
+    - molpx
+about:
+  home: https://github.com/markovmodel/molPX
+  license: LGPL v3+
+  # TODO: include in next release
+  #license_file: LICENSE.txt
```

Comment:
Hi guys, sorry to be late, I'm on a different timezone..just saw the PR and and will merge ASAP